src/Assetic/Filter/LessphpFilter.php

```diff
@@ -0,0 +1,165 @@
+<?php namespace October\Rain\Assetic\Filter;
+
+/*
+ * This file is part of the Assetic package, an OpenSky project.
+ *
+ * (c) 2010-2014 OpenSky Project Inc
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use October\Rain\Assetic\Asset\AssetInterface;
+use October\Rain\Assetic\Factory\AssetFactory;
+use October\Rain\Assetic\Util\LessUtils;
+
+/**
+ * Loads LESS files using the PHP implementation of less, lessphp.
+ *
+ * Less files are mostly compatible, but there are slight differences.
+ *
+ * @link http://leafo.net/lessphp/
+ *
+ * @author David Buchmann <david@liip.ch>
+ * @author Kris Wallsmith <kris.wallsmith@gmail.com>
+ */
+class LessphpFilter implements DependencyExtractorInterface
+{
+    private $presets = array();
+    private $formatter;
+    private $preserveComments;
+    private $customFunctions = array();
+    private $options = array();
+
+    /**
+     * Lessphp Load Paths
+     *
+     * @var array
+     */
+    protected $loadPaths = array();
+
+    /**
+     * Adds a load path to the paths used by lessphp
+     *
+     * @param string $path Load Path
+     */
+    public function addLoadPath($path)
+    {
+        $this->loadPaths[] = $path;
+    }
+
+    /**
+     * Sets load paths used by lessphp
+     *
+     * @param array $loadPaths Load paths
+     */
+    public function setLoadPaths(array $loadPaths)
+    {
+        $this->loadPaths = $loadPaths;
+    }
+
+    public function setPresets(array $presets)
+    {
+        $this->presets = $presets;
+    }
+
+    public function setOptions(array $options)
+    {
+        $this->options = $options;
+    }
+
+    /**
+     * @param string $formatter One of "lessjs", "compressed", or "classic".
+     */
+    public function setFormatter($formatter)
+    {
+        $this->formatter = $formatter;
+    }
+
+    /**
+     * @param boolean $preserveComments
+     */
+    public function setPreserveComments($preserveComments)
+    {
+        $this->preserveComments = $preserveComments;
+    }
+
+    public function filterLoad(AssetInterface $asset)
+    {
+        $lc = new \lessc();
+        if ($dir = $asset->getSourceDirectory()) {
+            $lc->importDir = $dir;
+        }
+
+        foreach ($this->loadPaths as $loadPath) {
+            $lc->addImportDir($loadPath);
+        }
+
+        foreach ($this->customFunctions as $name => $callable) {
+            $lc->registerFunction($name, $callable);
+        }
+
+        if ($this->formatter) {
+            $lc->setFormatter($this->formatter);
+        }
+
+        if (null !== $this->preserveComments) {
+            $lc->setPreserveComments($this->preserveComments);
+        }
+
+        if (method_exists($lc, 'setOptions') && count($this->options) > 0 ) {
```

Comment:
@mjauvin The code quality check is saying the issue lies on line 110:
https://github.com/octobercms/library/pull/485/checks#step:5:10
It's saying it expected 0 spaces *before* the closing bracket, but there is one there.

```suggestion
        if (method_exists($lc, 'setOptions') && count($this->options) > 0) {
```